internal/controller/standalone_pgadmin/users.go

```diff
@@ -264,7 +294,9 @@ cd $PGADMIN_DIR
 				log.Error(err, "PodExec failed: ")
 				continue
 			}
-			if strings.TrimSpace(stderr.String()) != "" {
+			if strings.Contains(strings.TrimSpace(stderr.String()), "UserWarning: pkg_resources is deprecated as an API") {
+				log.Info(stderr.String())
+			} else if strings.TrimSpace(stderr.String()) != "" {
 				log.Error(errors.New(stderr.String()), fmt.Sprintf("pgAdmin setup.py error for %s: ",
```

Comment:
Slightly out of scope for your work, but could you add something to differentiate between the add-user error here and the update-user error on line 264